internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -455,7 +521,7 @@ func (r *AgentServiceConfigReconciler) newAssistedServiceDeployment(instance *ad
 		{Name: "ISO_CACHE_DIR", Value: "/data/cache"},
 
 		// from configmap
-		{Name: "AUTH_TYPE", Value: "none"},
+		{Name: "AUTH_TYPE", Value: "local"},
```

Comment:
@carbonin @rwsu not sure if this was verified already but, will enabling `local` auth still allow Ironic to download the ISO image? or would the download call require authentication of some sort?

Reply:
The url in the AgentConfig will be signed with an auth token.